WordPress/Classes/Models/ReaderPost.h

```diff
@@ -15,6 +15,7 @@ extern NSString * const ReaderPostStoredCommentTextKey;
 @property (nonatomic, strong) NSString *authorDisplayName;
 @property (nonatomic, strong) NSString *authorEmail;
 @property (nonatomic, strong) NSString *authorURL;
+@property (nonatomic, strong) NSString *blavatar;
```

Comment:
Can I suggest calling this `siteIcon` (or better `siteIconURL`)? Not all the icons are blavatars anymore, the REST API uses that terminology, and in general things are moving towards "site icon"